bostrom/README.md

```diff
@@ -303,33 +291,32 @@ ETH (`capitalization_per_agent`).
 
 ## Predicting V Demand
 
-V are natural tokens. Each V enables its holder to produce 1 cyberlink daily. To simulate cyberlinks usage we
-have derived base estimate of `cyberlinks_per_day` formula from ETH data:
+V are natural tokens. Each V enables its holder to produce 1 cyberlink daily. To simulate cyberlink usage we have
+derived a base estimate of `cyberlinks_per_day` formula from ETH data:
 
 ![ETH Data](images/eth_transactions_per_active_address_regression.png)
 
-We derived such formula:
+We derived this formula:
 
 <p style="text-align:center;"><img src="https://render.githubusercontent.com/render/math?math=\color{green}transactions\_per\_agent = 9 \cdot agents\_count^{-0.3}"></p>
 
-And we adjusted such a formula by multiplication coefficient (`cyberlinks_trasactions_coeff`) because we expect
-that neurons in the BOSTROM network will be more active than agents in ETH. Also, we adjusted such a formula by
+Then we adjusted this formula by multiplication coefficient (`cyberlinks_trasactions_coeff`) because we expect
+that neurons in the BOSTROM network would be more active than agents in ETH. Also, we adjusted this formula by
 adding a number of `extra_links` and `guaranteed_links`.
 
 <p style="text-align:center;"><img src="https://render.githubusercontent.com/render/math?math=\color{green}cyberlinks\_per\_day = 9 \cdot cyberlinks\_trasactions\_coeff \cdot agents\_count^{-0.3} %2B extra\_links %2B guaranteed\_links"></p>
 
-
 `extra_links` count depends on UX specifics, such as setting the name of a neuron, following (proportion of
 neurons) and extra:
 
 <p style="text-align:center;"><img src="https://render.githubusercontent.com/render/math?math=\color{green}extra\_links ~ f( agents\_count, setting name, following, extra )">
 
-Also, the founding team envisions some basic stuff to cyberlink such as naming systems and tokens. So we can rely
-on this demand too.
+The founding team plans to create cyberlinks for blockchain naming systems and tokens to make them searchable. By
```

Comment:
@mastercyb check please. That was your text before but it was grammatically incorrect. So we tried to rephrase it, but probably the meaning might be damaged.